### PR TITLE
Issue4

### PR DESCRIPTION
**Pull request by immunity-anomaly**
implement fill_state.setter

### CODE REVIEW

File: vosekast_control/Scale.py [hunk 1]
```diff
@@ -207,9 +204,11 @@ def read_value_from_scale(self):
 
                 if splitted_line[0] == b'-':
                     self.logger.warning("Negative weight. Discarding value.")
+                    print(splitted_line)
```
Comment (karlTGA) [inline vosekast_control/Scale.py]: You should prefer the logger!

File: vosekast_control/Tank.py [hunk 2]
```diff
@@ -90,7 +91,7 @@ def prepare_to_fill(self):
                 self._state = self.BETWEEN
                 
                 #check if constant_tank full
-                while not self.vosekast.constant_tank.is_filled and self.fill_state == True:
+                while not self.vosekast.constant_tank.is_filled and self._fill_state == True:
```
Comment (karlTGA) [inline vosekast_control/Tank.py]:
```suggestion
                while not self.vosekast.constant_tank.is_filled and self._fill_state:
```

File: vosekast_control/Scale.py (hunk 1 above)
Comment (immunity-anomaly) [reply vosekast_control/Scale.py]: done.

File: vosekast_control/Tank.py (hunk 2 above)
Comment (immunity-anomaly) [reply vosekast_control/Tank.py]: done.

**Comment by immunity-anomaly** [comment]
fill_state is used to be able to stop the countdown in tank/fill function.

File: vosekast_control/TestSequence.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ def __init__(
                 self.logger.debug("Scale running, continuing.")
 
             # set fill to True
-            self.vosekast.constant_tank.fill_state = True
+            self.vosekast.constant_tank.state = "IS_FILLING"
```
Comment (karlTGA) [inline vosekast_control/TestSequence.py]: You can use here the propeties of the tank: 
```suggestion
            self.vosekast.constant_tank.state = self.vosekast.constant_tank.IS_FILLING
```

File: vosekast_control/TestSequence.py [hunk 4]
```diff
@@ -190,7 +190,7 @@ def pause_sequence(self):
 
         # set fill countdown to False
         for tank in self.vosekast.tanks:
-            tank.fill_state = False
+            tank.state = "BETWEEN"
```
Comment (karlTGA) [inline vosekast_control/TestSequence.py]: Is 'BETWEEN' still needed if you have `IS_FILLING`, `IS_DRAINING` and `STOPPED`.

File: vosekast_control/Tank.py [hunk 5]
```diff
@@ -15,7 +15,6 @@ class Tank():
     FILLED = 1
```
Comment (karlTGA) [inline vosekast_control/Tank.py]: That seems inconstist for me. You use strings and numbers for one state. 
I would suggest to use only strings. So `FILLED = 'FILLED'`.

Comment (immunity-anomaly) [reply vosekast_control/Tank.py]: ok

File: vosekast_control/TestSequence.py (hunk 4 above)
Comment (immunity-anomaly) [reply vosekast_control/TestSequence.py]: will be fixed in next commit

**Pull request merged**